state/session/service.js

```diff
@@ -0,0 +1,27 @@
+import { createLogic } from 'redux-logic';
+
+import { API_KEY } from '../../constants';
+import { SESSION_REQUEST_TOKEN } from './types';
+import { sessionRequestTokenSuccess } from './actions';
+
+const sessionRequestTokenLogic = createLogic({
+  type: SESSION_REQUEST_TOKEN,
+  warnTimeout: 0,
+
+  process({ httpClient }, dispatch, done) {
+    httpClient.get(`3/authentication/token/new?api_key=${API_KEY}`)
```

Comment:
It's better to move `3` to base url and api_key to common params in api client

Reply:
Fixed